cli/cmd/integration.go

```diff
@@ -92,7 +93,9 @@ var (
 		Short: "Show details about a specific external integration",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(_ *cobra.Command, args []string) error {
+			var resp api.V2CommonIntegration
```

Comment:
we don't need this variable until we are in `109`, let's move it there for clarify